.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+.vscode
```

Comment:
unnecessary.

Reply:
Added to not enter commits

Reply:
but these commits aren't there to be made!